ui/main.qml

```diff
@@ -480,6 +486,12 @@ StatusWindow {
                     console.error("!!! ONBOARDING FINISHED WITH ERROR:", error)
                     return
                 }
+
+                if (flow === Onboarding.OnboardingFlow.LoginWithLostKeycardSeedphrase) {
```

Comment:
I think the progress page push should not happen here but not sure about the correct place where to put it... :thinking: any ideas @micieslak ?

Reply:
I think this should happen earlier, inside OnboardingLayout or Flow; and then we don't emit the finished signal. The result is that we show the progress page and then Quit anyway right?

Reply:
Yeah, I agree that putting into `OnboardingLayout` is good idea. Ideally we should not modify the stack from there at all and keep the OnboardingLayout responsible for stack maintenance.

Reply:
Agreed to keep this solution (at least for now) as the simplest, despite its inconsistency.